app/kuma-cp/cmd/run.go

```diff
@@ -71,7 +72,16 @@ func newRunCmdWithOpts(opts runCmdOpts) *cobra.Command {
 				return err
 			}
 			runLog.Info(fmt.Sprintf("Current config %s", cfgBytes))
-			if cfg.Mode != config_core.Global {
+			switch cfg.Mode {
+			case config_core.Standalone:
+				if cfg.Mode != config_core.Local {
```

Comment:
Isn't this if always true?

Reply:
Moving code around mistake. Good catch. Thanks.